install/scripts/install-json-hpp.sh

```diff
@@ -1,21 +1,43 @@
 #!/bin/bash
 
+# Source fossology config file, for Proxy settings
+conf_file=/etc/fossology.conf
+[ -r $conf_file ] && . $conf_file
+
 cd "$(dirname "$0")/../.."
 FINAL_FILE=./src/copyright/agent/json.hpp
-SHASUM=fbdfec4b4cf63b3b565d09f87e6c3c183bdd45c5be1864d3fcb338f6f02c1733
-TMP=$(mktemp)
-CHECKSUMFILE=$(mktemp)
+FILE_SHASUM="fbdfec4b4cf63b3b565d09f87e6c3c183bdd45c5be1864d3fcb338f6f02c1733"
+FILE_URL="https://github.com/nlohmann/json/releases/download/v2.1.1/json.hpp"
```

Comment:
Please check the version as well. (v2.1.1 > v3.1.2)

Reply:
Stupid me, URL is also fixed.